service_taxi/src/main/kotlin/org/thechance/service_taxi/data/utils/Extensions.kt

```diff
@@ -1,6 +1,26 @@
 package org.thechance.service_taxi.data.utils
 
+import com.mongodb.client.model.Updates
+import com.mongodb.client.result.UpdateResult
+import org.bson.conversions.Bson
 import org.litote.kmongo.coroutine.CoroutineFindPublisher
+import kotlin.reflect.full.memberProperties
 
 fun <T : Any> CoroutineFindPublisher<T>.paginate(page: Int, limit: Int) =
-    skip((page - 1) * limit).limit(limit)
\ No newline at end of file
+    skip((page - 1) * limit).limit(limit)
+
+fun UpdateResult.isSuccessfullyUpdated() = modifiedCount > 0L
+
+fun Any?.isNotNull() = this == null
+
+fun <T> updateNotNullProperties(
+    obj: T,
+    otherFilter: (propertyName: String) -> Boolean = { true },
```

Comment:
I haven't got this function used for what, but the name of this parameter maybe :
```suggestion
    filters: (propertyName: String) -> Boolean = { true },
```